Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
Will this create issues with older Rust toolchains? When was version 4 support added?

Reply:
O.o have no idea, i just ran this today using cargo 
1.83.0 2024-10-29